test/compilable/extra-files/objdump-postscript.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+expect_file=${EXTRA_FILES}/${TEST_NAME}.out
+obj_file=${OUTPUT_BASE}_0.o
+
+echo Creating objdump
+objdump --disassemble --disassembler-options=intel-mnemonic "${obj_file}" > "${obj_file}.dump"
+
+echo SANITIZING Objdump...
+< "${obj_file}.dump" \
+    tail -n+3 | \
+    sed s/^.*section// > "${obj_file}.dump.sanitized"
```

Comment:
don't need the sed script anymore

Reply:
Yes, we do - to support FreeBSD.

Reply:
Removed it as the current solution doesn't support FreeBSD either :O